site/app/templates/admin/Extensions.twig

```diff
@@ -52,6 +52,7 @@
                             <option value="{{ reason }}">{{ reason }}</option>
                         {% endfor %}
                     </select>
+                    <script> registerSelect2Widget("reason-for-exception"); </script>
```

Comment:
I think we should place all the scripts in one area in the twig file instead of adding separate script tags in different places.

Reply:
That's fair. I was thinking I would put it there because it would be more clear to new developers wanting to implement select2 widgets that you need that script tag for it to work, but you're right that putting it with the other scripts is probably more organized.